docs/pages/docs/guides/auth-and-access-control.mdx

```diff
@@ -0,0 +1,532 @@
+import { ComingSoon } from '../../../components/docs/ComingSoon';
+import { Markdown } from '../../../components/Markdown';
+
+# Authentication and Access Control
+
+Keystone comes with several features that work together to control access to the Admin UI and GraphQL API:
+
+- **Session Management** – a set of APIs for starting and ending user sessions, as well as initialising the session data for each request ([docs](../apis/session))
+- **The `auth` package** – an opinionated implementation of authentication features for Keystone apps ([docs](../apis/auth))
+- **Access Control** – a powerful framework for restricting access to specific lists, operations, and fields ([docs](../apis/access-control))
+- **Dynamic UI Config and Field Modes** – options for the Admin UI that work similarly to Access Control, and let you dyamically configure the Admin UI based on user permissions ([docs](../apis/schema#ui))
+
+Session Management and Auth are extremely flexible in Keystone, and it's possible to replace the default implementations we provide with your own (or integrate an entirely separate auth system), but in this guide we'll focus on how all these features are designed to work together.
+
+## Setting up Users, Auth and Session
+
+Keystone is designed to make as few assumptions about your schema and system design as possible, which means it doesn't come with a built-in concept of **Users**. In your app, Users are just another List that you create and specify fields for – you can set them up however you like.
+
+### Create a list for users
+
+To use the `auth` package, you need to nominate a list that stores your user accounts, and that lists needs at least two fields: an **identity** field (e.g username or email address, it should be unique) that users are looked up by when they sign in; and a **secret** field (i.e password) that is used to verify them.
+
+You can add any other fields you want to your users list, including contact information, permissions, roles, and relationships to other entities in your database.
+
+Here's an example:
+
+```ts
+const Person = list({
+  fields: {
+    name: text(),
+    email: text({ isIndexed: 'unique' }),
+    password: password(),
+    isAdmin: checkbox(),
+  },
+});
+```
+
+!> Read more about creating lists in the [schema](../apis/schema) and [fields](../apis/schema) API Docs
+
+### Configure authentication
+
+With our users list in place, we can start configuring our authentication:
+
+```ts
+const { withAuth } = createAuth({
+  listKey: 'Person',
+  identityField: 'email',
+  secretField: 'password',
+});
+```
+
+The `createAuth` function returns another function called `withAuth` that will automatically extend Keystone's config to set up everything we need. Behind the scenes, the auth package is just using lower-level Keystone APIs to do everything, which means if you want to do something differently, you can fork our implementation of auth and build your own (this is true for session management as well)
```

Comment:
nit: We make the assumption here that the reader knows where `createAuth` comes from, and go straight into explaining what it does. We should probably at the minimum mention that its an export from `@keystone-next/auth`. It's implicitly stated in the second paragraph, and that information is available in the Auth API Docs link below, but it'd be nice to have that information explicitly declared and not hidden behind a click, at this point in the guide.

Reply:
Good catch @gwyneplaine 

I've added another paragraph to the section above, and put the import in the code snippet to make the import path clear.

For the rest I think it's safe to assume the reader already knows where to import `list` and field types from 🤞